src/main/resources/static/js/src/HistoryController.js

```diff
@@ -0,0 +1,36 @@
+const app = angular.module("myApp", []);
+app.controller("HistoryController", function($scope,$http) {
+    let myPhone = document.getElementById("from").textContent;
```

Comment:
use angular for this: https://codippa.com/how-to-get-value-of-input-box-in-angular/
https://codippa.com/how-to-get-value-of-input-box-in-angular/